Assets/MixedRealityToolkit.Examples/Demos/UX/BoundingBox/Scripts/BoundingBoxTest.cs

```diff
@@ -93,6 +93,7 @@ private IEnumerator Sequence()
 
                 SetStatus("Change target bounds override size");
                 bc.size = new Vector3(0.5f, 0.1f, 1);
```

Comment:
The expected behavior is for the bbox component to actually detect the change in the bounds size automatically, calling createrig will actually hide the real bug. Can you remove this and I'll file a bug that bbox doesn't change when the bounds changes?

Reply:
Filed #5414 to track fixing the bug